Source/MQTTnet.AspnetCore/ServiceCollectionExtensions.cs

```diff
@@ -21,13 +21,13 @@ public static IServiceCollection AddHostedMqttServer(this IServiceCollection ser
             return services;
         }
         
-        public static IServiceCollection AddHostedMqttServer(this IServiceCollection services, Action<MqttServerOptionsBuilder> configure)
```

Comment:
Is this a breaking change? Because the parameter is changed. If yes I would like to merge it for 2.9.0. Otherwise for 2.8.3.

Reply:
If it's a breaking change... you could possibly mark the existing one as `[Obsolete]` and still release as 2.8.3... and then folks need to change-up before [i think] 2.9 or would that be 3.0? (Trying to recall semver rules here)

Reply:
it breaks binary compatibility but the source is compatible, if this is considered breaking I can provide another overload of that method